lib/apsi-bug-tracker-stack.ts

```diff
@@ -33,5 +35,38 @@ export class APSIBugTrackerStack extends cdk.Stack {
     exampleRoute.addMethod('GET', exampleLambdaIntegration);
     const IDExampleRoute = exampleRoute.addResource('{id}');
     IDExampleRoute.addMethod('GET', exampleLambdaIntegration);
+	
+    // Create DB security group
+    const vpc = ec2.Vpc.fromLookup(this, 'VPC', {isDefault: true})
+	
+    const APSIBugTrackerSG = new ec2.SecurityGroup(this, 'ApsiBugTracker-sg', {
+      vpc,
+      allowAllOutbound: true,
+      description: 'security group for the APSIBugTracker',
+    });
+
+    APSIBugTrackerSG.addIngressRule(
+      ec2.Peer.anyIpv4(),
+      ec2.Port.allTraffic(),
+    );
+
+    APSIBugTrackerSG.addIngressRule(
+      ec2.Peer.anyIpv6(),
+      ec2.Port.allTraffic(),
+    );
+
+    // Create database instance
+    const instance = new rds.DatabaseInstance(this, 'Instance', {
+      engine: rds.DatabaseInstanceEngine.mysql({ version: rds.MysqlEngineVersion.VER_8_0_26 }),
+      instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.MICRO),
+      allocatedStorage: 20,
+      vpc,
+      vpcSubnets: {
+        subnetType: ec2.SubnetType.PUBLIC,
+      },
+      securityGroups: [APSIBugTrackerSG],
+      databaseName: 'apsidb',
+      backupRetention: cdk.Duration.days(0)
+    });
```

Comment:
Wspomniałeś, że credentiale trzymasz w aws'owym serwisie, czy to nie powinno być  gdzieś tu podpięte, bo baza chyba zostanie zainicjalizowana z defaultowymi credentialami, czy się mylę?

Reply:
Credentiale zostały podpięte.